### PR TITLE
Update logstash chart to support custom ports

### PR DESCRIPTION
**Pull request by ChiefAlexander**
- [ ] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [x] README.md updated with any new values or changes
- [ ] Updated template tests in `${CHART}/tests/*.py` 
- [ ] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

**Comment by ChiefAlexander** [comment]
I've signed the CLA. Not sure how to rerun the check...

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by ChiefAlexander** [comment]
@jmlrt Thanks for the review! I have made your suggested changes.

**Comment by jmlrt** [comment]
jenkins test this please

File: logstash/tests/logstash_test.py [hunk 1]
```diff
@@ -180,6 +180,20 @@ def test_adding_a_extra_container():
...
+extraPorts:
+  - name: foo
+    containerPort: 30000
+"""
+    r = helm_template(config)
+    extraPorts = r["statefulset"][name]["spec"]["template"]["spec"]["containers"][0]["ports"]
+    assert {
+        "name": "foo",
+        "containerPort": 30000,
+    } in extraPorts
```
Comment (jmlrt) [inline logstash/tests/logstash_test.py]:
```suggestion
    extraPorts = r["statefulset"][name]["spec"]["template"]["spec"]["containers"][0][
        "ports"
    ]
    assert {"name": "foo", "containerPort": 30000,} in extraPorts
```

**Comment by jmlrt** [comment]
jenkins test this please

**Pull request merged**